python_modules/libraries/dagster-dbt/dagster_dbt/dbt_manifest.py

```diff
@@ -7,13 +8,19 @@
 DbtManifestParam = Union[Mapping[str, Any], str, Path]
 
 
+@lru_cache(maxsize=None)
+def read_manifest_path(manifest_path: Path) -> Mapping[str, Any]:
+    return cast(Mapping[str, Any], orjson.loads(manifest_path.read_bytes()))
```

Comment:
Worth a comment explaining the cache? 

If we ever go back and fix the fact that the definition holds on to this instead of GCing it, we will go back and revert this i assume?